src/classes/application.js

```diff
@@ -160,6 +161,21 @@ module.exports = class Application extends Base {
     const app = express()
     _app.set(this, app)
 
+    log.trace('Ash server adding body parsing middleware')
+    const bodyParserOptions = config.bodyParser || {}
+    if (bodyParserOptions.json !== false) {
```

Comment:
Alright, so opt-out instead of opt-in, yeah? I was just a bit thrown off by the example then. I'm not sure I agree that it should be opt-out though... Seems unintuitive, because usually in general the presence of code means doing something, whereas no code does nothing.

The alternative opt-in strategy would be that the framework code itself is opt-in, but the app generator is opt-out in that all of the default options are set on the ENV when you scaffold out the project. Does that make sense?